configs/kintohub.json

```diff
@@ -1,9 +1,9 @@
 {
   "index_name": "kintohub",
   "start_urls": [
-    "https://docs.kintohub.com/docs/getting-started.html",
-    "https://docs.kintohub.com/docs/"
+    "https://docs.kintohub.com/docs/introduction/getting-started"
```

Comment:
```suggestion
    "https://docs.kintohub.com/docs/"
```

`start_urls` will be used as a regex to match the sitemap in this context. The broader it is, the better

Reply:
![image](https://user-images.githubusercontent.com/32097720/65586140-eeb41580-df83-11e9-8680-308b8030063a.png)
I can't do it on my own.

Reply:
ok, just fixed it